README.md

```diff
@@ -104,6 +104,9 @@ There's a known issue with WordPress `alloptions` cache design. Specifically, a
 
 ## Changelog ##
 
+### Latest ###
```

Comment:
```suggestion
### 1.4.1 (May 11, 2023) ###
```

Reply:
🤦🏻 change sitting there unsaved in my editor